### PR TITLE
set travis wait time to 25 mins

### PR DESCRIPTION
**Pull request by dray92**
Signed-off-by: Debosmit Ray <debo@uber.com>

### CODE REVIEW

**Comment by dray92** [comment]
How does this change affect these files?

**Comment by vprithvi** [comment]
It doesn't. However, the codecov only maintains the code coverage last
successful build. When there are build failures, the report becomes wacky.

**Comment by dray92** [comment]
@vprithvi @yurishkuro does this current change make more sense? I tried `--debug` but that prints more than 4mb in logs

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,10 @@ clean:
 test:
 	./gradlew check
 
+.PHONY: test-travis
+test-travis:
+	./gradlew -is check --info
```
Comment (vprithvi) [inline Makefile]: Why are you setting the `info` flag redundantly using `-i` and `--info` ?

**Pull request merged**